### ISSUE TITLE
Why don't firstVisibleIndex and lastVisibleIndex bind or notify?

### ISSUE THREAD

**Issue by jmbldwn** [PolymerElements/iron-list#347]
I have to actively read these properties with javascript, and there isn't a good way to get an event that would fire when these properties change, so I'm resorting to a timer to monitor them, which is pretty stupid.

If I could bind to these properties, I could have a declarative way of knowing what's visible.

Is this behavior intentional?  Is there another way to get notified when they change?

**Comment by blasten** [comment]
Yes. That declarative properties are desirable from an API perspective although it might have a cost for users that don't care about these notifications. Depending on how it's implemented, the updates could be debounced to reduce stress. I'll keep the issue open.

**Comment by davidmaxwaterman** [comment]
Is it possible to use these values to tell me when the view is 'full', or when iron-list starts to fill the view? At the moment, I have a progress bar to tell the user when the data is loading, but assigning that data to the iron-list results in a pause while the UI does nothing. I would like an event to tell me when iron-list is starting to fill the view so I can stop my 'progress-bar' (it's not needed any more since the user can see the view filling up).
Is there some other event I could use?
